strings/strings.json

```diff
@@ -7369,6 +7369,46 @@
     "en": "Remove filter on {0}",
     "fr": "Enlever le filtre sur {0}"
   },
+  "Select": {
```

Comment:
As explained, we will need a dedicated phrase for the "Select facet / counts", we cannot use this like that where we concatenate strings together like this.